scipy/stats/morestats.py

```diff
@@ -1083,6 +1100,19 @@ def boxcox_normmax(x, brack=(-2.0, 2.0), method='pearsonr'):
         'all'
             Use all optimization methods available, and return all results.
             Useful to compare different methods.
+    optimizer : callable, optional
+        `optimizer` is a callable that accepts one argument:
+
+        fun : callable
+            The objective function, which evaluates the negative
+            log-likelihood function at a provided value of `lmbda`
+
+        and returns an object, such as an instance of
+        `scipy.optimize.OptimizeResult`, which holds the optimal value of
+        `lmbda` in an attribute `x`.
```

Comment:
```suggestion
        fun : callable
            The objective function to be optimized. `fun` accepts one argument,
            the Box-Cox transform parameter `lmbda`, and returns the negative
            log-likelihood function at the provided value. The job of `optimizer`
            is to find the value of `lmbda` that minimizes `fun`.

        and returns an object, such as an instance of
        `scipy.optimize.OptimizeResult`, which holds the optimal value of
        `lmbda` in an attribute `x`.
```
Does this help @tupui?

Reply:
Yes clearer IMO, thanks!